lib/tasks/evm_dba.rake

```diff
@@ -171,6 +147,3 @@ namespace :evm do
     end
   end
 end
-
-Rake::Task["db:setup"].prerequisites.unshift("evm:db:db_setup_not_supported")
-Rake::Task["db:reset"].prerequisites.unshift("evm:db:db_setup_not_supported")
```

Comment:
This has saved us a countless number of times - not sure why we'd remove this.

Reply:
Yea. I understand why we wanted that before. Was hoping that `rake db:test:prepare` now worked out of the box. Will have to double check this

Reply:
Nick was trying to get standard schema repo methods working in https://github.com/ManageIQ/manageiq-schema/issues/502, but I don't think that ever landed.  The main thing that doesn't work through a "normal" db:setup is the metrics inheritance.

Reply:
This has been pulled out into another effort